src/crypto/DeviceList.js

```diff
@@ -318,6 +318,10 @@ export default class DeviceList {
                 this._sessionStore.storeEndToEndDevicesForUser(
                     userId, storage,
                 );
+
+                if (token) {
+                    this._sessionStore.storeEndToEndDeviceSyncToken(token);
```

Comment:
it's slightly different - the point is to store a token representing the time when we last *successfully* updated the device lists (which may be some time before the last successful sync) - so that when we resume, we can ask for the users which have been updated since then.

When @Kegsay's stuff lands, we'll need to watch out that we still do the `/changes` dance, even though we won't be doing an initial sync - because we don't have a record of which users were out-of-date at the point of exit. (Or we could keep a localstorage/indexeddb record of which users are out of sync.)